management/src/main/scala/akka/management/AkkaManagement.scala

```diff
@@ -100,25 +119,38 @@ final class AkkaManagement(implicit system: ExtendedActorSystem) extends Extensi
   }
 
   /**
-   * Get the routes for the HTTP management endpoint.
+   * Scala API: Get the routes for the HTTP management endpoint.
    *
    * This method can be used to embed the Akka management routes in an existing Akka HTTP server.
    */
   def routes: Try[Route] = prepareCombinedRoutes(providerSettings)
 
+  // FIXME should `routes` return `Try` of throw IllegalArgumentException?
```

Comment:
This is likely used on start up and I'd say it is irrecoverable, I wouldn't want my app to start up if the management routes hadn't been setup correctly. But if they try to use the route they'll find out the Try is failed.

I don't have a strong opinion either way

Reply:
done in https://github.com/akka/akka-management/pull/436